### PR TITLE
Buttons refactor and adding to the storybook

### PR DESCRIPTION
**Pull request by milosyuki**
Adjusted buttons to the design. Left only `filter`, `filterActive` and `category` variants because we don't have them in the design but do on website. Added to storybook.

### CODE REVIEW

**Comment by robwalsh21** [comment]
@milosyuki  This looks good.  Should we also include refactor of IconButton in this PR, as I can see some sizing issues with the footer?

**Review by thecotne — APPROVED** [review]

**Comment by milosyuki** [comment]
I have added `IconButton` component as well as `Button` `icon` variant, so the same style can be used in both cases. Please review once more @robwalsh21 @thecotne . Thanks, guys.

File: src/blocks/HomepageHero.tsx
Comment (robwalsh21) [inline src/blocks/HomepageHero.tsx:1]: Hey @milosyuki ,  There is a variant of buttons for the home hero in the figma file.  They are slightly different to the standard solid and outline buttons

Comment (milosyuki) [reply src/blocks/HomepageHero.tsx:1]: Ah, yes, I have created those two variants but forgot to use them in the right place. Thanks! Fixed.

**Review by robwalsh21 — APPROVED** [review]
Looks great @milosyuki

**Pull request merged**